app/assets/stylesheets/components/_space-addon.scss

```diff
@@ -1,3 +1,6 @@
+.mb-half { margin-bottom: 4px; }
```

Comment:
are these the same as `.mt-tiny` and `.mb-tiny`?

Reply:
yeah, `mb-tiny` and `mt-tiny` are .25rem which equates to 4px, so these can be removed.

Reply:
I took care of this (along with other things) here: https://github.com/18F/identity-idp/pull/2099/commits/f92628b73d26dddf8125d12da2edf7f0eecae267

More fixes will be coming later. The cancellation links, for example, aren't set up properly.